stackrc

```diff
@@ -259,7 +259,8 @@ WSGI_MODE=${WSGI_MODE:-"uwsgi"}
 
 # Base GIT Repo URL
 # Another option is https://git.openstack.org
-GIT_BASE=${GIT_BASE:-git://git.openstack.org}
+#GIT_BASE=${GIT_BASE:-git://git.openstack.org}
+GIT_BASE=${GIT_BASE:-https://git.openstack.org}
```

Comment:
This is an issue due to Intel's proxy, handle it in configuration by setting GIT_BASE in local.conf

Reply:
Ok, I will move it out.